spec/index.html

```diff
@@ -225,30 +301,29 @@
           attribute boolean? expanded;
           attribute boolean? disabled;
           attribute InvalidType? invalid;
-          attribute boolean hasPopUp;
+          attribute boolean? hasPopUp;
+          attribute boolean? pressed;
 
           // Control values.
-          attribute DOMString? value;
+          attribute DOMString? valueText;
           attribute DOMString? placeholder;
-          attribute double? rangeValue;
-          attribute double? rangeMin;
-          attribute double? rangeMax;
-          attribute double? rangeStep;
+          attribute double? valueNow;
+          attribute double? valueMin;
+          attribute double? valueMax;
 
           // Live regions.
-          attribute boolean? liveAtomic;
-          attribute boolean? liveBusy;
-          attribute LiveType? liveStatus;
-          attribute LiveRelevantType? liveRelevant;
+          attribute boolean? atomic;
+          attribute boolean? busy;
+          attribute LiveType? live;
+          attribute LiveRelevantType? relevant;
 
-          // Relationships.
+          // Other relationships.
           attribute AccessibleNode? activeDescendant;
           attribute iterable&lt;AccessibleNode&gt; controls;
-          attribute iterable&lt;AccessibleNode&gt; describedBy;
           attribute AccessibleNode? details;
           attribute AccessibleNode? errorMessage;
           attribute AccessibleNode? flowTo;
-          attribute iterable&lt;AccessibleNode&gt; labeledBy;
+          attribute iterable&lt;AccessibleNode&gt; owns;
```

Comment:
We decided that virtual nodes in phase 3 would hang off of the DOM tree,
i.e. the AccessibleNode for a DOM element could have virtual AccessibleNode
children, but not the other way around.

That solves a ton of complexity while still solving pretty much all of the
real-world use cases we thought of.

However, that seems to imply we should keep aria-owns...